tests/integration/helpers/integration-test-helper.js

```diff
@@ -136,7 +136,7 @@ TestUtility.prototype.VerifyFileContents = function (dir, fileName, expectedCont
     var _this = this;
     _this.runFunc(function () {
         var bundleContents = _this.FileSystem.readFileSync(dir + '/' + fileName, 'utf8');
-        expect(bundleContents).toBe(expectedContents);
+        expect(bundleContents.replace(/\s/g, '')).toBe(expectedContents.replace(/\s/g, ''));
```

Comment:
@Andrew-Hagedorn-ZocDoc This makes our integration tests ignore whitespace when comparing text. I got fed up with trying to make sure it all matched perfectly, it was really hard to find missing spaces in the jasmine test error messages.